apps/compartmentConverter.cpp

```diff
@@ -61,6 +63,19 @@ template< class T > void requireEqualCollections( const T& a, const T& b )
     REQUIRE_EQUAL( j, b.end( ));
 }
 
+/** @return true if the cell occupies a continuous region in the report frame */
+bool _isContinuous( const brion::CompartmentReport& report,
+                    const size_t gidIndex )
+{
+    const auto& offsets = report.getOffsets()[gidIndex];
+    const auto& counts = report.getCompartmentCounts()[gidIndex];
+    for( size_t i = 0; i < offsets.size() - 1; ++i )
+        if( offsets[i] + counts[i] != offsets[i+1] )
+            return false;
```

Comment:
Since this is an optimization, a false negative is not a concern. I'll see if I find an easy implementation for your comment.